src/Tickets/Blocks/Tickets/app/editor/controls/template.js

```diff
@@ -44,16 +45,32 @@ RadioInput.propTypes = {
 	onProviderChange: PropTypes.func,
 };
 
-const Controls = ({
+/**
+ * Get the block controls for the Tickets block.
+ *
+ * @since TBD
+ *
+ * @param {Object}   props                      The component properties.
+ * @param {boolean}  props.disabled             Whether the controls should be disabled.
+ * @param {boolean}  props.hasMultipleProviders Whether there are multiple providers.
+ * @param {Node}     props.message              The message to display.
+ * @param {Function} props.onProviderChange     The function to call when the provider changes.
+ * @param {Array}    props.providers            The available providers.
+ * @param {string}   props.selectedProvider     The selected provider.
+ *
+ * @return {Array} The block controls.
+ */
+function getTicketsBlockControls({
 	disabled,
 	hasMultipleProviders,
 	message,
 	onProviderChange,
 	providers,
 	selectedProvider,
-}) =>
-	hasMultipleProviders && (
-		<InspectorControls key="inspector">
+}) {
+	const controls = [];
+	if (hasMultipleProviders) {
```

Comment:
```suggestion
	if ( hasMultipleProviders ) {
```

Reply:
This is JS, not PHP - we don't put in the spaces. I'm not sure why we have two different formatting styles but we currently do.